mcp-server/coneshare_mcp/config.py

```
@@ -0,0 +1,30 @@
+    mcp_host: str
+    mcp_port: int
+    mcp_path: str
+
+
+def get_settings() -> Settings:
+    # Smart default: inside docker use http://backend:8000/api/v1, outside use http://localhost:8000/api/v1
+    default_url = "http://backend:8000/api/v1"
+    if "CONESHARE_API_URL" not in os.environ:
+        try:
+            socket.gethostbyname("backend")
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)